easybuild/easyconfigs/h/HDF/HDF-4.2.13-intel-2017a-no-netcdf.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'HDF'
+version = '4.2.13'
+versionsuffix = '-no-netcdf'
```

Comment:
Since the other easyconfigs don't have this either, I wouldn't include this...

Instead, it would be better to enable netCDF (& Fortran) support?

Reply:
See above comment.  This is needed per gdal documentation.